resources/js/visual-editor/blocks/accordion/accordion.css

```
@@ -0,0 +1,116 @@
+    justify-content: space-between;
+    padding-right: 2.5rem;
+}
+
+.ap-accordion__title-content {
+    flex: 1;
+    cursor: pointer;
+    padding: 0.75rem 1rem;
+}
+
+.ap-accordion__title-content:focus-visible {
+    outline: 2px solid currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)